src/setup.js

```diff
@@ -0,0 +1,143 @@
+import StackdriverErrorReporter from 'stackdriver-errors-js'
```

Comment:
Is this the file that does all the setup when a network changes and we re-run this when something does change?

Reply:
Note: let's go through this part in more detail over a call

Reply:
`stackdriver-errors-js` can be removed now that @LeonmanRolls added different log report library